spec/Body/BackupLights.vspec

```diff
@@ -0,0 +1,12 @@
+# (C) 2022 BMW Group - All rights reserved.
+# AUTHOR: Gerrit Grotewold BMW Group;
+
+IsOn:
+  datatype: boolean
+  type: actuator
+  description: Indicates if light is on or off. True = On. False = Off.
```

Comment:
This uses the singular, but the file name uses the plural. Consider renaming the file.

For some types of lights that are blinking, this terminology can be confusing. Hazrds lights, for example, blink on and off. Is this signal intended to be sent every time the light comes on and goes off? Probably not.

Hence, perhaps this should be renamed to IsActivated?

Reply:
thank you. your feedback is part of latest commit.